pages/dkp/kommander/1.4/clusters/index.md

```diff
@@ -7,70 +7,87 @@ menuWeight: 7
 excerpt: View clusters created with Kommander or any connected Kubernetes cluster
 ---
 
-Kommander allows you to monitor and manage very large numbers of clusters, so it's easy to either connect existing clusters or create new clusters whose lifecycle is managed by Konvoy.
+Kommander allows you to monitor and manage very large numbers of clusters. Use the features described in this area of the product to connect existing clusters, or to create new clusters whose lifecycle is managed by Konvoy. You can view clusters from the Clusters tab in the navigation pane on the left. You can see the details for a cluster by selecting the **View Details** link at the bottom of the cluster card or the cluster name in either the card or the table view.
```

Comment:
More of a style question - should we use "described in this area" instead of "described in this area of the product"?

```suggestion
Kommander allows you to monitor and manage very large numbers of clusters. Use the features described in this area to connect existing clusters, or to create new clusters whose lifecycle is managed by Konvoy. You can view clusters from the Clusters tab in the navigation pane on the left. You can see the details for a cluster by selecting the **View Details** link at the bottom of the cluster card or the cluster name in either the card or the table view.
```

Reply:
Dropped "of the product" as noted.